### DISCUSSION TITLE
Trapdoor doenst work with PlayerBlockInteractEvent

### DISCUSSION THREAD

**Discussion by DasLixou** [Minestom/Minestom#878] [Help]
hello!

My Code:
```java
eventHandler.addListener(PlayerBlockInteractEvent.class, event -> event.setCancelled(true));
```

but when right-clicking a trapdoor, it still opens

**Comment by Eoghanmc22** [comment]
although this change is client side (the trap door doesnt actually open), you need to set event.setBlockingItemUse() as well to make the server resend the chunk. setCancelled only prevents the placement rule from running.

**Reply by DasLixou** [reply]
Yeah but it doesnt behave well. It takes more time than Spigot and i can fall trought, in spigot i would get lagged back. How can i achieve this? i know this isnt notchian but does it work?

**Reply by MrlnHi** [reply]
I would suggest creating an issue for not getting lagged back
